ResearchKit/Common/ORKFormStep.h

```diff
@@ -129,6 +129,14 @@ ORK_CLASS_AVAILABLE
 @property (nonatomic, copy, readonly) NSString *identifier;
 
 /**
+ A Boolean value indicating whether the form item is optional.
+ 
+ The default value of this property is `NO`. When the value is `YES`, this form item doesn't need
```

Comment:
Yes, the only detail is that even if you set all *form items* as non-optional, they can still be overridden by using the *Skip button* if the *form step* is optional. This part is equivalent to what happened before.

But yeah, maybe setting the *form item optional default* value to `YES` is more homogenous, I'll do just that.